fdbclient/fdb_client.go

```diff
@@ -100,10 +279,61 @@ type mockFdbLibClient struct {
 	mockedError error
 	// requestedKey will be the key that was used to call getValueFromDBUsingKey.
 	requestedKey string
+	// coordinationState represents the current coordination state.
```

Comment:
There's a bit of a naming collision here because we also use "coordination state" in other contexts to refer to the state stored by the coordinators in the core database. I don't have a better alternative in mind, but we may need to limit how widely we use this combination of words.